modules/CountlyClass.js

```diff
@@ -3445,6 +3452,7 @@ class CountlyClass {
          * Check and send the events to request queue if there are any, empty the event queue
          */
         function sendEventsForced() {
+            processAsyncQueue(); // process async queue before sending events
```

Comment:
this is the wrong place to put this in.
It is a totally underlated operation from "sendEventsForced"

why does the "process all async operations assigned to the SDK" have to be tied into the "package all events into a request" call?